### PR TITLE
Allow the use of custom adapters

### PR DESCRIPTION
**Pull request by Isigiel**
## Summary
Allows the use of custom adapters with axios

## Description
Add info that custom adapters are supported which are then passed through to the sdk-core

## Motivation and Context
Allow the use of the contentful sdk in a server side rendered angular application

## Todos
Merge https://github.com/contentful/contentful-sdk-core/pull/67 to make this work

-   [x] Implemented feature
-   [ ] Feature with pending implementation

## Screenshots (if appropriate):
Not needed

### CODE REVIEW

**Comment by Isigiel** [comment]
Okay
@mohamedturki this should be ready to go I updated the dependency too

File: README.md [hunk 1]
```diff
@@ -156,6 +156,10 @@ const contentful = require("contentful");
 const contentful = require("contentful/dist/contentful.browser.min.js");
 ```
 
+#### Angular universal:
```
Comment (Khaledgarbaya) [inline README.md]: Can you move this section under advanced?

Comment (Isigiel) [reply README.md]: Alright, I took the react stuff too and moved them both over

**Comment by Isigiel** [comment]
I took both angular  and react specifics and moved them into the advanced.md

**Comment by Khaledgarbaya** [comment]
hey @Isigiel this should've been released automatically but travis ci somehow failed for no reason, I restarted the build and it should be out now.
Sorry for the trouble, computers 🤷‍♀️

**Pull request merged**